interval/lift/lift.go

```diff
@@ -132,3 +134,66 @@ func StrictBorderCheck(c chain.Chain, i interval.Interval) bool {
 	_, border = chain.TPosToQPos(c, i.GetChromEnd()-1) //interval ranges are open right so we want ChromEnd - 1
 	return border
 }
+
+//everything below is for lifting with AXT
+
+// refCoordToRefIdx is a helper function for LiftCoordinatesWithAxt which identifies the start and end indices in the Axt reference multifa record that corresponds to the interval to be lifted
+func refCoordToRefIdx(a axt.Axt, region interval.Interval) (start, end int) {
```

Comment:
Note to self: this function is finding how "ref pos in target" translates to "aln pos in query", taking gaps into account